### PR TITLE
new(ci): added update-kernels gha.

### PR DESCRIPTION
**Pull request by cappellinsamuele**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area CI

**Does this PR require a change in the driver versions?**

No

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Porting of update-kernels Prow job.
Waiting for Poiana token (missing at the moment).

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/update-kernels.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Update Kernels
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '* * * * 2'
```
Comment (FedeDP) [inline .github/workflows/update-kernels.yml]: This should be a weekly job, like: `0 2 * * 1` (ie: At 02:00 on Monday.)

Comment (cappellinsamuele) [reply .github/workflows/update-kernels.yml]: Got it, thanks

**Review by FedeDP — APPROVED** [review]
/approve

**Pull request merged**